ci/infra/testrunner/skuba/skuba.py

```diff
@@ -89,6 +94,26 @@ def node_bootstrap(self, cloud_provider=None, timeout=None):
 
         self.checker.check_node("master", 0, stage="joined", timeout=timeout)
 
+    def _setup_container_registries(self, registry_mirror):
+        mirrors = {}
+        for l in registry_mirror:
+            if l[0] not in mirrors:
+                mirrors[l[0]] = []
+            mirrors[l[0]].append(l[1])
+
```

Comment:
So this implies that I can add multiple mirrors for a single registry, right? I am not sure how useful this is, but should it at least be documented in the help?

Reply:
My understanding is that is possible in the cri-o configuration, but it is not actually implemented in this PR (maybe we could implement in a future PR allowing multple occurrences of the `--registry-mirror` parameter.

Reply:
AFAIK it will work in this PR. But I haven't tested it.

Reply:
Done.